api/office_structure/models.py

```diff
@@ -17,4 +19,18 @@ class OfficeStructure(Base, Utility, BaseNestedSets):
     )
 
     def __repr__(self):
-        return "<{}>".format(self.name)
+        return "<OfficeStructure {}>".format(self.name)
+
+    def add_node(self, name, parent_id=None):
```

Comment:
This function has not been tested according to circle-ci. Is there a way you could test this?

Reply:
Yes @bevkololi , there is away and I have added a test for them. Thanks for the pointer